packages/chat_repository_interface/lib/src/interfaces/pending_message_repository_interface.dart

```diff
@@ -0,0 +1,38 @@
+import "package:chat_repository_interface/src/models/message_model.dart";
+
+/// The present chat repository interface
```

Comment:
please change chat everywhere to message because chat means something different in this flutter_chat package.
Unless we allow for presending entire chats.